components/signup/InterestsPage.tsx

```diff
@@ -41,7 +41,7 @@ export default ({ student }): JSX.Element => {
       </Box>
       <Box className={classes.inner}>
         <Typography className={classes.title}>Areas of Interest</Typography>
-        <ArrayForm label="Project Topics" items={interests} setItems={setInterests} />
+        <ArrayForm label="Project Topics" items={interests} setItems={setInterests} options={interests} />
```

Comment:
- need to add options prop in line 30, then pass that into ArrayForm's options
- modify pages/signup/setup to fetch from backend (like you did the filter pages)